estate/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is not necessary in Python files for a long time already.
There are some that are still there in the code for historical reason, but please don't add them on new files :slightly_smiling_face: 
AFAIK, the licensing is also not really necessary to add on each file since it's defined in the manifest for the whole module.